Issue compiling tests

From a fresh checkout of the project, I just encountered the following issue in running the unit tests with default settings:

```
error[E0790]: cannot call associated function on trait without specifying the corresponding `impl` type
   --> src/xml_db/parse/meta.rs:456:32
    |
456 |         assert_eq!(value.uuid, Default::default());
    |                                ^^^^^^^^^^^^^^^^^^ cannot call associated function of trait
    |
help: use a fully-qualified path to a specific available implementation
    |
456 |         assert_eq!(value.uuid, </* self type */ as Default>::default());
    |                                +++++++++++++++++++        +

error[E0283]: type annotations needed
   --> src/xml_db/parse/meta.rs:456:9
    |
456 |         assert_eq!(value.uuid, Default::default());
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ cannot infer type
    |
    = note: multiple `impl`s satisfying `Uuid: PartialEq<_>` found in the `uuid` crate:
            - impl PartialEq for Uuid;
            - impl PartialEq<NonNilUuid> for Uuid;
    = note: this error originates in the macro `assert_eq` (in Nightly builds, run with -Z macro-backtrace for more info)
```

--- a/src/xml_db/parse/meta.rs
+++ b/src/xml_db/parse/meta.rs
@@ -350,7 +350,7 @@ mod parse_meta_test {
         xml_db::parse::{parse_test::parse_test_xml, XmlParseError},
     };
 
-    use uuid::uuid;
+    use uuid::{uuid, Uuid};
 
     #[test]
     fn test_meta() -> Result<(), XmlParseError> {
@@ -453,7 +453,7 @@ mod parse_meta_test {
     #[test]
     fn test_custom_icon() -> Result<(), XmlParseError> {
         let value = parse_test_xml::<Icon>("<Icon></Icon>")?;
-        assert_eq!(value.uuid, Default::default());
+        assert_eq!(value.uuid, Uuid::default());
         assert_eq!(value.data.len(), 0);
 
         let value = parse_test_xml::<Icon>(

